DevGuide.md

```diff
@@ -1,5 +1,27 @@
 # Developer's Guide
 
+## References
+
+- [Catalog of Dependencies](#catalog-of-dependencies)
+- [Install Development and Build Tools](#install-development-and-build-tools)
+- [Setup Source Repository](#setup-source-repository)
+- [Setup Build Dependency Repository](#setup-build-dependency-repository)
+  * [Automatic script instructions](#automatic-script-instructions)
+  * [Manual download instructions](#manual-download-instructions)
+- [Building Ghidra](#building-ghidra)
+- [Developing Ghidra](#developing-ghidra)
+  * [Prepare the Environment](#prepare-the-environment)
+  * [Import Eclipse Projects](#import-eclipse-projects)
+  * [Building the natives](#building-the-natives)
+  * [Pre-compile Language Modules](#pre-compile-language-modules-optional)
+  * [Import and Build GhidraDev project](#import-and-build-ghidradev-project-optional)
+  * [Run/Debug Ghidra from Eclipse](#run/debug-ghidra-from-eclipse)
```

Comment:
I get an error on the "/".  Changing it to a "-" seems to make it happy.

Reply:
I can just fix this in a 2nd commit if you'd like.

Reply:
In GitHub it want `#rundebug-ghidra-from-eclipse` (without `/` at all). I prefer having Github experience.

Reply:
Does it not work with the -?  If not, then I agree we should go with what GitHub likes.

Reply:
Unfortunately does not work. But I have better idea. Let's change to "Run and Debug Ghidra from Eclipse" that way it would not be an issue and we can make both Eclipse and GitHub happy

Reply:
Ok, good idea.  I think that is how I solved this problem once before :)

Reply:
Done.